Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * ABI: Include events in the ABI that are emitted by but not defined inside a contract.
```

Comment:
I see that this PR changes `ContractDefinition::interfaceEvents` so should have an effect on the ABI, but why don't we have any test which shows that?

Reply:
Because we did not have a test yet that would have shown it. The events in https://github.com/ethereum/solidity/pull/10996/files#diff-41f735f63a07b2011f8b288c3a7d2567667261e1b423263d6f08993d0b0c0074R61 are not listed prior to this PR.